.github/workflows/node.yml

```diff
@@ -177,3 +177,44 @@ jobs:
             - name: Test compatibility
               run: npm test -- -t "set and get flow works"
               working-directory: ./node
+
+
+    build-linux-musl-latest:
+        runs-on: ubuntu-latest
+        container:
+            image: node:18.19.0-alpine
+        timeout-minutes: 15
+        steps:
+            - name: Install bash
+              run: |
+                apk add bash
+                apk add gettext
+
+            - name: Install git
+              run: apk add git
+
+            - uses: actions/checkout@v4
+
+            - name: Checkout submodules
+              run: |
+                  git config --global --add safe.directory "$GITHUB_WORKSPACE"
+                  git submodule update --init --recursive
+
+            - name: Build Node wrapper
+              uses: ./.github/workflows/build-node-wrapper
+              with:
+                  os: "ubuntu-latest-musl"
+                  target: "aarch64-unknown-linux-musl"
+                  github-token: ${{ secrets.GITHUB_TOKEN }}
+                  arch: "x64"
+                  named_os: "linux"
+
+            - name: Create a symbolic Link for redis6 binaries
```

Comment:
Do we need to step in the alpine image? it's relevant to amazon-linux which installs redis as redis6